site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -184,7 +184,7 @@
       "source": [
         "### Get an example dataset\n",
         "\n",
-        "To demonstrate how to save and load weights, you'll use the [MNIST dataset](http://yann.lecun.com/exdb/mnist/). To speed up these runs, use the first 1000 examples:"
```

Comment:
I think the commas here and across the docs in general serve a purpose and should not be removed. Check out [Commas after introductory words and phrases](https://developers.google.com/style/commas?hl=en#commas-after-introductory-words-and-phrases) (Google developer docs style guide).

Reply:
It sounds a little strange to me, but I think it's fine with the commas so I added them back in in almost all cases.